airflow/utils/db.py

```diff
@@ -78,6 +78,19 @@ def merge_conn(conn, session=None):
         session.commit()
 
 
+@provide_session
+def add_default_pool_if_not_exists(session=None):
+    from airflow.models.pool import Pool
+    if not Pool.get_pool(Pool.DEFAULT_POOL_NAME, session=session):
+        default_pool = Pool(
+            pool=Pool.DEFAULT_POOL_NAME,
+            slots=128,
```

Comment:
Is it worth making people's upgrades easier and reading the old config value (if it's set) here?

Something like:

```suggestion
            slots=config.get('core', 'non_pooled_task_slot_count', fallback=128),
```

Reply:
Good idea.